addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -616,70 +616,26 @@ function getCSSRules(doc) {
                 }
             }
             for (const subRule of subRules) {
-                const selectorText = subRule.selectorText;
-                if (selectorText && !SELECTORS_IGNORE.test(selectorText)) {
-                    const style = _normalizeStyle(subRule.style);
-                    if (Object.keys(style).length) {
-                        for (let selector of selectorText.split(',')) {
-                            selector = selector.trim();
+                const selectorText = subRule.selectorText || '';
```

Comment:
This one has a similar issue. What about people who would have the update code but the old xml and thus still have duplicated rules ? How will this commit impact the performances in this case ? I suggest to test it properly by removing the xml fix from the previous commit locally then try with and without this commit. If the performances with this commit are worse then it might be betterto delay it to master since the whole point of this PR is to improve performances.